Liber Usualis - mei3/liberupdatev5.py

```diff
@@ -0,0 +1,239 @@
+import lxml.etree as ET
+
+def main(filename: str) -> None:
+    """
+    Converts an MEI file to MEI 5.0 format.
+
+    Args:
+        filename: The path to the MEI file to be converted.
+    """
+    ET.register_namespace("mei", "http://www.music-encoding.org/ns/mei")
+
+    tree = ET.parse(filename)
+    root = tree.getroot()
+
+    # Dictionaries for mapping attribute values
+    div_dict = {"minor": "maior", "minior": "maior", "major": "maxima", "final": "finalis", "small": "minima", "comma": "virgula"}
+    epi_dict = {"horizontal": "h", "vertical": "v"}
+
+    # Flags for tracking certain conditions
+    theres_an_episema = False
+    theres_a_pb = False
+    theres_a_sb = False
+    flag = False
+    neume_flag = False
+    neume_flag = False
+
+    # Extra attributes for neumes and notes
+    extra_nc = {}
+    extra_neume = {}
+
+    # Set MEI version to 5.0
+    root.attrib["meiversion"] = "5.0"
+
+    # Iterate over all elements in the tree
+    for child in root.iter("*"):
+        child.tag = ET.QName(child).localname
+
+        # Handle graphic elements (e.g., change href to target)
+        if child.tag.endswith("graphic"): #changes some stuff in the headers
+            for att in list(child.attrib):
+                if att.endswith("href"):                    
+                    child.attrib['target'] = child.attrib.pop(att)
```

Comment:
It seems counterintuitive to me to iterate through the `child` attributes to get a single attribute?

Why not `child.get("href")` (maybe you have to namespace it)?

Reply:
iteration is used to avoid problems with namespace

Reply:
Are the namespaces of the liber MEI 3 files inconsistent?

Reply:
Sometimes the namespace shows up, sometimes it doesn't

Reply:
Like, different mei3 files will have different namespaces? 

Do you have an example?

Reply:
I just ran a script to print out every unique attribute of the files. One attribute has {http://www.w3.org/1999/xlink} as a namespace, one has {http://www.w3.org/XML/1998/namespace}, the remaining have no namespace. I can look into removing the iteration for attributes with no namespace